### DISCUSSION TITLE
AndroidAuto and 2.5D view

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#17168] [Q&A]
Hello folks,

All my test shows that connected to AndroidAuto, OSMandMe lost the 2.5D view, and I always had the "flat" rendering map.

Is it coming from OSMAndMe, my car or AndroidAuto?

Thanks to take a little bit of your time to answer :-)

**Comment by vshcherb** [comment]
Yes we have an open issue for that but we have problems to enable OpenGL engine on Android Auto still especially we don't how to get it working when phone screen is off

**Comment by ghost** [comment]
Oh! Thanks for your clear answer. Could your gibe me the PR #? Then I could read all your advance on this issue. Thanks in advance and one a again, Good job girls, boys and other,
